Formula/structurizr-cli.rb

```diff
@@ -0,0 +1,25 @@
+class StructurizrCli < Formula
+  desc "Command-line utility for Structurizr"
+  homepage "https://structurizr.com"
+  url "https://github.com/structurizr/cli/releases/download/v1.4.4/structurizr-cli-1.4.4.zip"
+  sha256 "ac173013c397fd2180a2b8134e29f8dfd3e0884418252433bcc9982e15b03909"
+  license "Apache-2.0"
+
+  bottle :unneeded
+
+  depends_on "openjdk"
+
+  def install
+    jar = "structurizr-cli.jar"
+    libexec.install "structurizr-cli-#{version}.jar" => jar
+    (bin/"structurizr-cli").write <<~EOS
```

Comment:
We have a convenience function for this (I believe it's `write_jar_wrapper`) please use that.

Reply:
Not really `write_jar_wrapper` but I've found the solution looking in *similar* formulae. Thanks for the suggestion.